src/main/java/uk/gov/companieshouse/overseasentitiesapi/model/updatesubmission/changelist/changes/beneficialowner/psc/IndividualBeneficialOwnerPsc.java

```diff
@@ -12,9 +12,6 @@ public class IndividualBeneficialOwnerPsc extends Psc {
   @JsonProperty("nationalityOther")
   private String nationalityOther;
 
-  @JsonProperty("birthDate")
```

Comment:
If we're removing this then we need to update the AC on the ticket

Reply:
done